src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,155 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+
+Code generated by Microsoft (R) PSSwagger
+Changes may cause incorrect behavior and will be lost if the code is regenerated.
+#>
+
+<#
+.SYNOPSIS
+    Create a migration job for managed disks.
+
+.DESCRIPTION
+    Create a disk migration job.
+
+.PARAMETER ResourceId
+    The resource id.
+
+.PARAMETER Disks
+    The parameters of disk list.
+
+.PARAMETER TargetShare
+    The target share name.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER Name
+    The migration job guid name.
+
+.PARAMETER InputObject
+    The input object of type Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob.
+
+.EXAMPLE
+
+    PS C:\> $disks = Get-AzsDisk -location local
+    PS C:\> New-AzsDiskMigrationJob -Name "MyMigrationJob" -Disks $disks -location local -TargetShare "\\SU1FileServer.azurestack.local\SU1_ObjStore"
+#>
+function New-AzsDiskMigrationJob
+{
+    [OutputType([Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob])]
+    [CmdletBinding(DefaultParameterSetName='DiskMigrationJobs_Create')]
+	[Alias('Start-AzsDiskMigrationJob')]
+    param(    
```

Comment:
Remove anything related to ParameterSets.

Reply:
Sure